docs/userguide/networking/work-with-networks.md

```diff
@@ -0,0 +1,454 @@
+
+In addition to the `--subnetwork` option, you also specify the `--gateway` `--ip-range` and `--aux-address` options.
+
+```bash
+docker network create -d overlay
+  --subnet=192.168.0.0/16 --subnet=192.170.0.0/16
+  --gateway=192.168.0.100 --gateway=192.170.0.100
+  --ip-range=192.168.1.0/24
+  --aux-address a=192.168.1.5 --aux-address b=192.168.1.6
+  --aux-address a=192.170.1.5 --aux-address b=192.170.1.6
+  my-multihost-newtork
+```
+
+Be sure that your subnetworks do not overlap. If they do, the network create fails and Engine returns an error.
+
+## Connect containers
+
+Docker containers can dynamically connect to one or more networks. These
+networks can be backed the same or different network drivers. Once connected,
+the containers can communicate using only another container's IP address or
+name.
+
+For `overlay` networks or custom plugins that support multi-host
+connectivity, containers connected to the same multi-host network but launched
+from different Engines can also communicate in this way.
+
+Create two containers for this example:
+
+```bash
+$ docker run -itd --name=container1 busybox
+18c062ef45ac0c026ee48a83afa39d25635ee5f02b58de4abc8f467bcaa28731
+
+$ docker run -itd --name=container2 busybox
+498eaaaf328e1018042c04b2de04036fc04719a6e39a097a4f4866043a2c2152
+```
+
+Then create a isolated, `bridge` network to test with.
+
+```bash
+$ docker network create -d bridge isolated_nw
+f836c8deb6282ee614eade9d2f42d590e603d0b1efa0d99bd88b88c503e6ba7a
+```
+
+Connect `container2` to the network and then `inspect` the network to verify the connection:
+
+```
+$ docker network connect isolated_nw container2
+$ docker network inspect isolated_nw
+[[
+    {
+        "Name": "isolated_nw",
+        "Id": "f836c8deb6282ee614eade9d2f42d590e603d0b1efa0d99bd88b88c503e6ba7a",
+        "Scope": "local",
+        "Driver": "bridge",
+        "IPAM": {
+            "Driver": "default",
+            "Config": [
+                {}
+            ]
+        },
+        "Containers": {
+            "498eaaaf328e1018042c04b2de04036fc04719a6e39a097a4f4866043a2c2152": {
+                "EndpointID": "0e24479cfaafb029104999b4e120858a07b19b1b6d956ae56811033e45d68ad9",
+                "MacAddress": "02:42:ac:15:00:02",
+                "IPv4Address": "172.21.0.2/16",
+                "IPv6Address": ""
+            }
+        },
+        "Options": {}
+    }
+]
+```
+
+
+You can see that the Engine automatically assigns an IP address to `container2`. If you had specified a `--subnetwork` when creating your network, the network would have used that addressing. Now, start a third container and connect it to the network on launch using the `docker run` command's `--net` option:
```

Comment:
I'd swap the examples; run the first one with `--net=`, and the second one using dynamically connecting it to the network